src/coord/src/session/vars.rs

```diff
@@ -66,6 +66,12 @@ const INTEGER_DATETIMES: ServerVar<bool> = ServerVar {
     description: "Reports whether the server uses 64-bit-integer dates and times (PostgreSQL).",
 };
 
+const QGM_OPTIMIZATIONS: ServerVar<bool> = ServerVar {
+    name: static_uncased_str!("qgm_optimizations"),
```

Comment:
If this will eventually go away, it should be `qgm_optimizations_experimental`.

Reply:
Done.